core/store.go

```diff
@@ -32,7 +32,7 @@ func init() {
 	store = make(map[unsafe.Pointer]*Obj)
 	expires = make(map[*Obj]uint64)
 	keypool = make(map[string]unsafe.Pointer)
-	WatchChannel = make(chan WatchEvent, 100)
+	WatchChannel = make(chan WatchEvent, 10000000)
```

Comment:
Let's make the buffer size configurable as well, for now we can reuse the `KeysLimit` config (we can improve this later).

Reply:
Used the same variable `config.keyLimit` for the watchchannel buffer.